bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHolmesHandler.java

```diff
@@ -117,18 +119,67 @@ public void initialize() {
     public void dispose() {
         logger.debug("WemoHolmesHandler disposed.");
 
-        ScheduledFuture<?> job = refreshJob;
+        ScheduledFuture<?> job = this.pollingJob;
         if (job != null && !job.isCancelled()) {
             job.cancel(true);
         }
-        refreshJob = null;
+        this.pollingJob = null;
         removeSubscription();
+        UpnpIOService localService = service;
+        if (localService != null) {
+            localService.unregisterParticipant(this);
+        }
```

Comment:
Why? This is already done by `removeSubscription()` called from the line above?

Reply:
done